### DISCUSSION TITLE
is it normal dupeguru indicate 100% match when file don't have same size with the reference ?

### DISCUSSION THREAD

**Discussion by iostrym** [hsoft/dupeguru#1110] [Q&A]
with scan type : filename
application mode : standart

it is very quick to look for dupe in this mode. only the filename is checked.

but I think I could focus on match percentage to look fo exact dupe : identical name + identical size give me a good confidence.

but I discover that the tool put in the list some file with different sizes. (less than 1%) it is dangerous because I could have deleted all these whereas they are not dupplicate.

is it normal ? is there a mode as quick as this one that could help me not to remove file that have a size that is different from the reference ?

**Comment by madias123** [comment]
Yes, this makes Dupeguru not useable.

**Comment by krautho** [comment]
Yes, if I want to compare the filenames, then I want to get all 100% matching filenames, whatever size they have. Otherwise I wouldn't compare filenames. Look at all the possibilities and options.

**Comment by aziouk** [comment]
why are you not using dupe guru's content mode, which actually will hash and compare contents reliably, yes, what krautho said
